### DISCUSSION TITLE
Graphics improvement suggestions

### DISCUSSION THREAD

**Discussion by Lurkars** [Lurkars/gloomhavensecretariat#40] [Ideas]
As already  discusses in #37, I create a new discussion about all improvements for graphics.

In general the assets from the creator pack are not High-Res, so polished version would be fine.

On my personal side, I really would like have a more generic [stats template](https://github.com/Lurkars/gloomhavensecretary/blob/main/src/assets/images/monster/stats/standard.png) without the icons and lines. So it can be adapted to Monsters without range and I can add sharp icon. And of course waiting for all Frosthaven assets, especially the boss stats.

**Comment by Infi220** [comment]
I just checked monster stat cards in my physical game and all of them have the same 4 icons in the same order, regardless of their abilities. If a monster doesn't have a given ability, there is a dash instead of a number on the stat card. I suggest to keep this exactly the same, so it's consistent with the physical game. Consistency is important in the app which has very tiny icons on smalls screen. If we were to change the order of the icons players can easily make mistakes. 

For example, they expect a movement stat in the 2nd row but Ancient Artillery doesn't have that ability, so the stat card has a dash. If the app were to hide that line and instead show a melee attack in the 2nd row, a player may think it's still movement because they are used to seeing that info in that spot. Plus, small icons are very hard to read. A dash makes it clear that given ability is not available. I suggest to keep as is.

**Reply by Lurkars** [reply]
To make it exact to the game is what I want here and in JOTL and Frosthaven there is nor range in stats, see for example here: https://github.com/any2cards/worldhaven/raw/master/images/monster-stat-cards/jaws-of-the-lion/jl-black-imp-0.png

**Comment by Infi220** [comment]
I see, thanks for clarifying. I see that when an icon is unavailable, the other icons are larger to fill the space. This looks neat. 

My previous observation was based on GH. I was not aware in JOTL and FH it is different.

I will work on this asset and post here when I'm done.

**Comment by Infi220** [comment]
How about this? Besides removing the icons and lines, I also removed that gray diagonal line on the right side (it was a leftover shadow from gray background), and I slightly blended the two bottom 45 degree corners of the golden line because the joints didn't match .
![standard2](https://user-images.githubusercontent.com/34073985/202971251-077bcf1c-8738-43ea-a060-982dcfbfa12e.png)

**Reply by Lurkars** [reply]
Wow awesome, thank you so much, exact what I was looking for. So you mind doing the same for the boss stats? 🙃

**Comment by Infi220** [comment]
Lol, sure, I'll give it a try :)

**Comment by Infi220** [comment]
Not sure if the center line between white and red is supposed to be straight or jagged. In the source it looks jagged but if so, then it's odd that standard stat cards are straight. Anyway, give it a try and see if it looks good or needs any adjustments. I also fixed the gray shadow and the two bottom 45 degree joints.
![boss2](https://user-images.githubusercontent.com/34073985/202991249-53343547-8605-473f-92dd-7f4c5df2a301.png)

**Comment by Lurkars** [comment]
Looks really awesome. I hope you're happy with my first implementation in [v0.26.6](https://github.com/Lurkars/gloomhavensecretary/releases/tag/v0.26.6). Can of course be improved a lot, but had to fix some other stuff in the meantime.

**Comment by Infi220** [comment]
Yes, it looks nice! I noticed 1 bug and 1 regression though:
1) When you create only an elite monster the stat sheet has 4 icons even if the monster has 3 stats. If I add regular monster, the sheet now changes to 3 icons.
2) When the sheet has only 3 icons the font for numbers is much larger. This causes a problem if numbers are >= 10  because they don't fit anymore. I think stat numbers don't need to be so large compared to the rest of the text, so I suggest using the same font size in 3-icon sheets as you use in 4-icon sheets.
![Untitled-1](https://user-images.githubusercontent.com/34073985/203189127-34031460-278f-4bd6-915d-d2d605aeeb73.jpg)

**Reply by Lurkars** [reply]
1. Is on purpose, if you has hide stats enabled, it should not spoil of range or not, so I decided to show all icons. Edit: ah I see, it's also when only one stats not shown. Okay will fix 

2. Font size seemed higher to me on the 3 stats view, but all just guessing from pictures. So I will decrease size anyway.

**Reply by Lurkars** [reply]
Fixed.
